base/dates/periods.jl

```diff
@@ -373,99 +373,98 @@ Base.isequal(x::Period, y::CompoundPeriod) = isequal(y, x)
 Base.isequal(x::CompoundPeriod, y::CompoundPeriod) = x.periods == y.periods
 
 # Capture TimeType+-Period methods
-(+)(a::TimeType,b::Period,c::Period) = (+)(a,b+c)
-(-)(a::TimeType,b::Period,c::Period) = (-)(a,b-c)
-(+)(a::TimeType,b::Period,c::Period,d::Period...) = (+)((+)(a,b+c),d...)
-(-)(a::TimeType,b::Period,c::Period,d::Period...) = (-)((-)(a,b-c),d...)
+(+)(a::TimeType, b::Period, c::Period) = (+)(a, b + c)
+(-)(a::TimeType, b::Period, c::Period) = (-)(a, b - c)
+(+)(a::TimeType, b::Period, c::Period, d::Period...) = (+)((+)(a, b + c), d...)
+(-)(a::TimeType, b::Period, c::Period, d::Period...) = (-)((-)(a, b - c), d...)
```

Comment:
there's some strange associativity here too... if `a - b - c` is `a - (b - c)`, then shouldn't `a - b - c - d` be `a - (b - (c - d))` ?